add desc for clustering coefficient algorithm (#22)
vesoft-inc/nebula-algorithm: README.md

|    GraphTriangleCount    | network structure and tightness analysis|
|   BetweennessCentrality  | important node digging, node influence calculation|
|        DegreeStatic      | graph structure analysis|

> |   ClusteringCoefficient  | recommended, telecom fraud analysis|

Reviewer: ```suggestion
|   ClusteringCoefficient  | recommendation system, telecom fraud analysis|
```

   val prResult = PageRankAlgo.apply(spark, data, prConfig, false)
   ```
   

>    If your vertex id is String type, [Pagerank Example](https://github.com/vesoft-inc/nebula-algorithm/blob/master/example/src/main/scala/com/vesoft/nebula/algorithm/PageRankExample.scala) gives the encoding and decoding process for String id.

Reviewer: ```suggestion
   If your vertex ids are Strings, see [Pagerank Example](https://github.com/vesoft-inc/nebula-algorithm/blob/master/example/src/main/scala/com/vesoft/nebula/algorithm/PageRankExample.scala) for how to encoding and decoding them.
```

>     For other algorithms, please refer to [examples](https://github.com/vesoft-inc/nebula-algorithm/tree/master/example/src/main/scala/com/vesoft/nebula/algorithm)

Reviewer: ```suggestion
    For examples of other algorithms, see [examples](https://github.com/vesoft-inc/nebula-algorithm/tree/master/example/src/main/scala/com/vesoft/nebula/algorithm)
```
